src/core/expr/fexpr_mean.cc

```diff
@@ -81,7 +81,7 @@ class FExpr_Mean : public FExpr_ReduceUnary {
 
       return is_grouped? std::move(col)
                        : Column(new Latent_ColumnImpl(new Mean_ColumnImpl<T>(
-                           std::move(col), gby
+                           std::move(col), stype, gby
```

Comment:
We don't need to pass `stype` to `Mean_ColumnImpl`. It is the `Mean_ColumnImpl` that needs to pass `stype` to `ReduceUnary_ColumnImpl` constructor and it already knows that the `stype` is simply `col.stype()`.

Reply:
i dont understand - `reduce_unary` requries an SType:

```py
template <typename T_IN, typename T_OUT>
class ReduceUnary_ColumnImpl : public Virtual_ColumnImpl {
  protected:
    Column col_;
    Groupby gby_;
    

  public:
    ReduceUnary_ColumnImpl(Column &&col, SType stype, const Groupby& gby)
      : Virtual_ColumnImpl(gby.size(), stype),
        col_(std::move(col)),
        gby_(gby)
    {
      xassert(col_.can_be_read_as<T_IN>());
    }


    ColumnImpl *clone() const override {
      return new ReduceUnary_ColumnImpl(Column(col_), this->stype(), Groupby(gby_));
    }
```
using the stype allows passing either the col's stype, or in the case of count, pass SType::INT64

Reply:
>i dont understand - reduce_unary requries an SType

Yes, but you can get it from `col.stype()` inside of `Mean_ColumnImpl`.